src/executors/default.yml

```diff
@@ -9,6 +9,19 @@ parameters:
         description: >
             Pick a specific cimg/node image version tag:
             https://hub.docker.com/r/cimg/node
+    resource_class:
```

Comment:
just to keep things consistent, i think the type should be moved to the top, and i think the enum values should be indented

Reply:
Done 👍